tests/runtime/index.ts

```diff
@@ -121,8 +132,10 @@ function imports(abi: () => Abi, host: Host): WebAssembly.Imports {
       'log.log': (level: number, message: Pointer) => {
         host.log.log(level, readStr(message))
       },
-      'store.get': todo,
-      'store.set': todo,
+      'store.get': () => 0,
+      'store.set': (entity: Pointer, id: Pointer, data: Pointer) => {
```

Comment:
How did you get the information of this required interface from the wasm binary?

Reply:
Mostly from looking at the AssemblyScript function declarations https://github.com/graphprotocol/graph-ts/blob/73054c382676f5e9723c52845fe730fd954baacd/index.ts#L11-L12

And taking a look at the Wasm function types:
```
$ wasm2wat build/BatchExchange/BatchExchange.wasm | grep 'import "index" "store.get"'
  (import "index" "store.get" (func $~lib/@graphprotocol/graph-ts/index/store.get (type 2)))
$ wasm2wat build/BatchExchange/BatchExchange.wasm | grep 'type (;2;)'                
  (type (;2;) (func (param i32 i32) (result i32)))
$ wasm2wat build/BatchExchange/BatchExchange.wasm | grep 'import "index" "store.set"'
  (import "index" "store.set" (func $~lib/@graphprotocol/graph-ts/index/store.set (type 4)))
$ wasm2wat build/BatchExchange/BatchExchange.wasm | grep 'type (;4;)'                
  (type (;4;) (func (param i32 i32 i32)))
```

So for `store.get`, for example, it takes `(entity: string, id: string) -> Entity?`, so each `i32` parameter is a pointer to the entity name, and the ID string values. The host import is then expected to return a pointer to an `Entity` result.

Then to make sure I got the ABI right, its a combination of `console.log`-ing reads from the pointer and looking at https://github.com/graphprotocol/graph-node/tree/master/runtime/wasm Rust implementation.